src/videojs.ads.js

```diff
@@ -155,11 +160,43 @@ var
         src: player.currentSrc(),
         currentTime: player.currentTime(),
         type: player.currentType()
-      };
+      },
+      tracksPlayingHandler;
+
+    tracksPlayingHandler = function() {
+      var trackEls = this.el().querySelector('.vjs-tech').querySelectorAll('track'),
+          track,
+          i = trackEls.length,
+          j = 0;
+
+      while (i--) {
+        track = trackEls[i].track;
+
+        for (j = 0; j < suppressedTracks.length; j++) {
+          if (suppressedTracks[j].track === track) {
+            suppressedTracks[j].mode = track.mode;
+          }
+        }
+
+        track.mode = 'disabled';
```

Comment:
We do currently set the remote text tracks to disabled anyway. Enabling and disabling this in the transitions in and out of linear ad mode seems fine.

We probably should add any tracks that aren't already on the `suppressedTracks` list into that list for restoration. The `suppressedTracks` list is originally made from remote text tracks but it isn't necessarily the case that all the tracks would be available in that list that we want to disable. We wouldn't want any cues to fire while an ad is playing. I've been thinking that maybe this function is how we should be handling all the text tracks: loop through all the tracks available from `textTracks()` and also anything we can get via the `querySelectorAll('track')` (since not everything will necessarily show up in `textTracks()` until vjs5) and update the snapshot on every `adplaying`.
Though, maybe that refactor is a bit too ambitious for this PR/fix.